src/platform/site-wide/side-nav/components/NavItem.js

```diff
@@ -0,0 +1,98 @@
+// Dependencies
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { get } from 'lodash';
+// Relative
+import DuplicateLineLabel from './DuplicateLineLabel';
+import ExpandCollapseButton from './ExpandCollapseButton';
+import LabelText from './LabelText';
+
+const NavItem = ({
+  depth,
+  item,
+  index,
+  renderChildItems,
+  sortedNavItems,
+  toggleItemExpanded,
+}) => {
+  // Derive the item properties.
+  const expanded = get(item, 'expanded');
+  const id = get(item, 'id');
+  const isSelected = get(item, 'isSelected');
+  const label = get(item, 'label', '');
+
+  // Derive the depth booleans.
+  const isFirstLevel = depth === 1;
+  const isDeeperThanSecondLevel = depth >= 2;
+
+  // Caclculate the indentation for the child items.
+  const indentation = isDeeperThanSecondLevel ? 20 * (depth - 1) : 20;
+
+  // Determine if we should show a line at the end.
+  const showLine = isFirstLevel && index !== sortedNavItems.length - 1;
+
+  return (
+    <li className={`va-sidenav-level-${depth}`} key={id}>
+      <div
+        aria-label={label}
+        className={classNames({
+          'va-sidenav-item-label': true,
+          selected: isSelected,
+        })}
+        onClick={toggleItemExpanded(id)}
```

Comment:
Not right now, but I'm still thinking we're gonna need to convert this div with the onclick into an anchor or button tag. If it passes accessibility though, maybe not. I think we can do an `axeCheck` from React unit tests to be sure.

Reply:
Axe didn't complain when I tested it actually! Also, we can't have a button within a button because then axe does complain (which we would have in this case because of the expand/collapse button being within the clickable nav item row). 😬

Reply:
@kelsonic I think that this has to do with being able to have focus.  anchors can have focus whereas divs cannot.  TMK you cannot tab down to divs but you can with anchors.  I agree this should probably be an anchor.

Reply:
Ah darn it, it looks like `a` tags can't be nested under another `a` tag too just like buttons:

![image](https://user-images.githubusercontent.com/12773166/68956468-1e1e1c00-0796-11ea-8a19-c1950265e681.png)

Reply:
So basically what's happening right now is this:
```jsx
<div className="nav-item-row" onClick={onClick}>
  <a href="/asdf">Nav Item Label</a>
</div>
```
This is what you guys were proposing:
```jsx
<a className="nav-item-row" onClick={onClick}>
  <a href="/asdf">Nav Item Label</a>
</a>
```
but that threw the invalid dom nesting error 😭

Reply:
Yeah, that isn't valid HTML, but the behavior is weird. Does the design spec for this call for that `Nav Item Label` to be a link? I took it as just the label for the toggle functionality, like an accordion.

Reply:
I'll confirm but that's what I recall @rtwell saying!

Reply:
This is now updated to a `button` or an `a` tag depending on if there are child nav items 👍